### PR TITLE
Bancor FQT rollup

### PR DESCRIPTION
**Pull request by mzhu25**
## Description
Adds `MixinBancor` to the `BridgeAdapter`. 

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Review by hysz — APPROVED** [review]
Nice work!

File: contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinBancor.sol [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            // Grant an allowance to the Bancor Network.
+            LibERC20TokenV06.approveIfBelow(
+                IERC20TokenV06(path[0]),
+                bancorNetworkAddress,
+                sellAmount
+            );
+        }
+
+        // Convert the tokens
+        boughtAmount = IBancorNetwork(bancorNetworkAddress).convertByPath.value(payableAmount)(
```
Comment (dorothy-zbornak) [inline contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinBancor.sol:97]: Wait, why does this work? Shouldn't it be `convertByPath{value: ...}` in 0.6?

Comment (mzhu25) [reply contracts/zero-ex/contracts/src/transformers/bridges/mixins/MixinBancor.sol:97]: uhhh good question 🤔  
I guess the old syntax still works in 0.6? I'll change this

**Comment by dorothy-zbornak** [comment]
Looks good on simbot.

```bash
SELL WETH->USDC 9.52 -> 5922.11 ($5933.32) after 0.4s @ prod
	✔ PASS 9.52 -> 5922.11 ($5899.83) ($14.29)
	Bancor: 100%
	gas: 500378
SELL WETH->USDC 9.52 -> 5922.11 ($5933.32) after 0.4s @ dev
	✔ PASS 9.52 -> 5922.11 ($5901.35) ($12.77)
	Bancor: 100%
	gas: 447217
SELL DAI->USDC 15651.08 -> 15561.31 ($15651.08) after 6.3s @ prod
	✔ PASS 15651.08 -> 15561.31 ($15524.60) ($15.73)
	Bancor: 100%
	gas: 550862
SELL DAI->USDC 15651.08 -> 15561.31 ($15651.08) after 6.3s @ dev
	✔ PASS 15651.08 -> 15561.31 ($15525.44) ($14.89)
	Bancor: 100%
	gas: 521305
SELL USDC->DAI 20326.05 -> 20045.62 ($20298.65) after 8.5s @ prod
	✔ PASS 20326.05 -> 20045.62 ($20029.17) ($16.45)
	Bancor: 100%
	gas: 575916
SELL USDC->DAI 20326.05 -> 20045.62 ($20298.65) after 8.5s @ dev
	✔ PASS 20326.05 -> 20045.62 ($20030.06) ($15.56)
	Bancor: 100%
	gas: 544904
SELL USDT->WETH 471.75 -> 0.75 ($471.75) after 0.8s @ prod
	✔ PASS 471.75 -> 0.75 ($454.64) ($15.50)
	Bancor: 100%
	gas: 529235
SELL USDT->WETH 471.75 -> 0.75 ($471.75) after 0.8s @ dev
	✔ PASS 471.75 -> 0.75 ($456.26) ($13.88)
	Bancor: 100%
	gas: 473848

```

**Review by dorothy-zbornak — APPROVED** [review]

**Pull request merged**